lang-ca.js

```diff
@@ -1187,29 +1219,31 @@ SnapTranslator.dict.ca = {
 
     // math functions
     'abs':
-        'abs',
+        'valor absolut',
     'floor':
         'part entera',
     'sqrt':
         'arrel quadrada',
     'sin':
-        'sin',
+        'sinus',
```

Comment:
Translations like these seem a bit odd to me. "sin" is the universal mathematical symbol for the sine function, AFAIK. (If it isn't what's used in Catalan mathematical works, though, please say so. :))